src/core/mantle/atoms/compute.ts

```diff
@@ -157,6 +142,48 @@ export function computeAtom(cache?: typeof globalDataFeaturesCache) {
     await set(compute, undefined);
   });
 
+  const writeComputedFeatures = atom(
+    null,
+    async (get, set, value: { feature: Feature[]; computed: ComputedFeature[] }) => {
+      const currentLayer = get(layer);
+      if (currentLayer?.type !== "simple" || !currentLayer.data) return;
+
+      if (process.env.NODE_ENV === "development") {
```

Comment:
```suggestion
      if (import.meta.env.DEV) {
```

https://ja.vitejs.dev/guide/env-and-mode.html